app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -443,6 +450,19 @@ class MainActivity : BaseActivity() {
         }
     }
 
+    private fun handleInAppReview() = with(localSettings) {
+        if (showAppReviewDialog && appLaunches != 0 && appLaunches % 50 == 0) {
```

Comment:
Is it wanted to be that hard to show this view ? 
Because if the user doesn't exactly launch the app on the % 50th, and just come back from background, it won't be shown
So virtually this could never be presented